src/mumble/PulseAudio.cpp

```diff
@@ -334,9 +334,9 @@ void PulseAudioSystem::eventCallback(pa_mainloop_api *api, pa_defer_event *) {
 		bool do_stop        = false;
 		bool do_start       = false;
 
-		if ((!pai || !g.s.doEcho()) && (est == PA_STREAM_READY)) {
+		if ((!pai || g.s.echoOption == EchoCancelOptionID::DISABLED) && (est == PA_STREAM_READY)) {
 			do_stop = true;
-		} else if (pai && g.s.doEcho()) {
+		} else if (pai && g.s.echoOption == EchoCancelOptionID::DISABLED) {
```

Comment:
By following the logic of the old code, I think this should be
```suggestion
		} else if (pai && g.s.echoOption != EchoCancelOptionID::DISABLED) {
```

Reply:
Replacing these things is tedious and prone to error...

Reply:
Yes indeed :sweat_smile: